.pre-commit-hooks.yaml

```diff
@@ -1,34 +1,36 @@
 -   id: tmt-lint
     name: tmt lint
-    entry: bash -c "git ls-files --error-unmatch $(python3 -c 'import tmt; print(tmt.Tree(logger=tmt.Logger.create(), path=\".\").root)')/.fmf/version && tmt lint --failed-only --source $@" PAD
-    files: '.*\.fmf$'
+    # Use a simple wrapper instead of simply `tmt lint` in order to reorder
+    # some arguments that need to be passed to `tmt`
+    entry: python -m tmt._pre_commit --pre-check lint --failed-only --source
+    files: '(?:.*\.fmf|.*/\.fmf/version)$'
     verbose: false
     pass_filenames: true
     language: python
     language_version: python3
 
 -   id: tmt-tests-lint
     name: tmt tests lint
-    entry: bash -c "git ls-files --error-unmatch $(python3 -c 'import tmt; print(tmt.Tree(logger=tmt.Logger.create(), path=\".\").root)')/.fmf/version && tmt tests lint --failed-only --source $@" PAD
-    files: '.*\.fmf$'
+    entry: python -m tmt._pre_commit --pre-check tests lint --failed-only --source
+    files: '(?:.*\.fmf|.*/\.fmf/version)$'
     verbose: false
     pass_filenames: true
     language: python
     language_version: python3
 
 -   id: tmt-plans-lint
     name: tmt plans lint
-    entry: bash -c "git ls-files --error-unmatch $(python3 -c 'import tmt; print(tmt.Tree(logger=tmt.Logger.create(), path=\".\").root)')/.fmf/version && tmt plans lint --failed-only --source $@" PAD
-    files: '.*\.fmf$'
+    entry: python -m tmt._pre_commit ---pre-check plans lint --failed-only --source
```

Comment:
A typo reported by tests: `---pre-check` vs `--pre-check`.

Reply:
That is weird, especially since other step in there seems to run the pre-commit fine and it's only the tests section. I'll take a look at it

Reply:
Aaah, it's triple dashed there... man I'm blind. I was looking at the word trying to understand what typo I've made, if I used a different dash etc.